homeassistant/components/heos/__init__.py

```diff
@@ -246,17 +248,19 @@ def update_ids(self, mapped_ids: dict[int, int]):
 class GroupManager:
     """Class that manages HEOS groups."""
 
-    def __init__(self, hass, controller):
+    def __init__(self, hass, controller, players):
```

Comment:
Can we type this?

Reply:
Added! It started to turn into a mypi typing adventure.